prod/westeurope/internal/api/cosmosdb/container_service-preferences/terragrunt.hcl

```diff
@@ -0,0 +1,33 @@
+dependency "cosmosdb_account" {
+  config_path = "../account"
+}
+
+dependency "cosmosdb_database" {
+  config_path = "../database"
+}
+
+#Internal
+dependency "resource_group" {
+  config_path = "../../../resource_group"
+}
+
+# Include all settings from the root terragrunt.hcl file
+include {
+  path = find_in_parent_folders()
+}
+
+terraform {
+  source = "git::git@github.com:pagopa/io-infrastructure-modules-new.git//azurerm_cosmosdb_sql_container?ref=v3.0.3"
+}
+
+inputs = {
+  name                = "service-preferences"
+  resource_group_name = dependency.resource_group.outputs.resource_name
+  account_name        = dependency.cosmosdb_account.outputs.name
+  database_name       = dependency.cosmosdb_database.outputs.name
+  partition_key_path  = "/fiscalCode"
+
+  autoscale_settings = {
+    max_throughput = 50000
```

Comment:
Maybe this value is too high.
Is it an high impacted collection?
I suggest to start at 4000 (minimum) and increase the value to 10000 before rollout.

Reply:
I don't have enough data to make this choice. As a rule of thumb, this collection will be read as much as `services`, so I thought it could share the same configuration. 

But be free to set a lower value if you think is better.

Reply:
I'll set to 10000 for both (services and service-preference)